reV/batch/batch.py

```diff
@@ -88,12 +101,12 @@ def _parse_config(config):
                 # append the unique dictionary representation to the attr
                 arg_combs.append(comb_dict)
                 file_sets.append(s['files'])
-                set_tags.append(s.get('set_tag', ''))
+                set_tags.append(set_tag)
 
         return arg_combs, file_sets, set_tags
 
     @staticmethod
-    def _tag_value(value):
+    def _fix_tag_w_year(value):
```

Comment:
On line 124, doesn't parse_year have a bool option, making the try except redundant?

Reply:
Oh yeah! Maybe it didnt when i wrote this. Simplified in a new commit.